clouddriver-ecs/src/test/java/com/netflix/spinnaker/clouddriver/ecs/cache/EcsClusterCacheClientTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2017 Lookout, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.ecs.cache;
+
+import com.netflix.spinnaker.cats.cache.DefaultCacheData;
+import com.netflix.spinnaker.clouddriver.ecs.cache.client.EcsClusterCacheClient;
+import com.netflix.spinnaker.clouddriver.ecs.cache.model.EcsCluster;
+import com.netflix.spinnaker.clouddriver.ecs.provider.agent.EcsClusterCachingAgent;
+import org.junit.Test;
+import spock.lang.Subject;
+
+import java.util.Collections;
+import java.util.Map;
+
+import static com.netflix.spinnaker.clouddriver.ecs.cache.Keys.Namespace.ECS_CLUSTERS;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+public class EcsClusterCacheClientTest extends CommonCacheClient {
```

Comment:
Would tend to recommend Spock over straight up junit & mockito. It's more consistent with the rest of the project and will cut down your boilerplate. While we don't encourage usage of Groovy in `main`, we're totally down with continued Groovy use in `test`.

Reply:
@robzienert can the tests be removed _for now_, and be added in a separate pull.
Currently have them all in java/junit.

Reply:
We can merge as-is with the tests as they are (if the PR is ready), then double back for Spock... but definitely don't want to add code without tests at all.

Reply:
Sounds good 👍